### DISCUSSION TITLE
Cannot install in fresh Laravel installation google/apiclient error

### DISCUSSION THREAD

**Discussion by sergiodk5** [spatie/laravel-google-calendar#214] [Q&A]
I have tried to install the package several times in different machines. I always get the same error in terminal:
$ composer require spatie/laravel-google-calendar
Using version ^3.3 for spatie/laravel-google-calendar
./composer.json has been updated
Running composer update spatie/laravel-google-calendar
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - spatie/laravel-google-calendar[3.3.0, ..., 3.3.1] require google/apiclient ^2.2 -> satisfiable by google/apiclient[v2.2.0, ..., 2.x-dev (alias of dev-master)].
    - google/apiclient 2.x-dev is an alias of google/apiclient dev-master and thus requires it to be installed too.
    - spatie/laravel-google-calendar v3.x-dev requires illuminate/support ^5.4.0 -> found illuminate/support[v5.4.0, ..., 5.8.x-dev] but these were not loaded, likely because it conflicts with another require.
    - google/apiclient[dev-master, v2.6.0, ..., v2.10.1] require guzzlehttp/psr7 ^1.2 -> found guzzlehttp/psr7[1.2.0, ..., 1.x-dev] but the package is fixed to 2.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - google/apiclient[v2.2.0, ..., v2.2.4] require monolog/monolog ^1.17 -> found monolog/monolog[1.17.0, ..., 1.x-dev] but the package is fixed to 2.3.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - google/apiclient[v2.3.0, ..., v2.5.0] require guzzlehttp/guzzle ~5.3.1||~6.0 -> found guzzlehttp/guzzle[5.3.1, ..., 5.3.x-dev, 6.0.0, ..., 6.5.x-dev] but it conflicts with your root composer.json require (^7.0.1).
    - Root composer.json requires spatie/laravel-google-calendar ^3.3 -> satisfiable by spatie/laravel-google-calendar[3.3.0, 3.3.1, v3.x-dev].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

**Comment by larawellcome** [comment]
I have same issue!
Solution:
1st install composer require guzzlehttp/psr7 ^1.9 (version number is important),
2nd install: composer require google/apiclient
3rd install: composer require spatie/laravel-google-calendar

**Reply by sergiodk5** [reply]
Thank you for you answer.
I managed to make it work by deleting the composer.lock file and install it again. it will worked for me.
